### DISCUSSION TITLE
Face recognition. Wrongly mixed 2 people with same name

### DISCUSSION THREAD

**Discussion by woho213** [photoprism/photoprism#3040] [General]
Hello,
Yesterday I wrongly mixed 2 people, in face recognition, with the same name. Each one had more than 2500 photos correctly recognized and indexed. Now I have only one person with more than 5k pictures
How can I solve this issue?
Changing the name one picture at a time seems like a tedious workflow
I do not want to fully reescan all the library, it could take ages (100k photos). Any one has other solutions? Is there a ctrl z?

Thanks

**Comment by lastzero** [comment]
How did you do that? Merging two persons into one is possible, but you should see a confirmation dialog.

In any case, face recognition works with face clusters, not individual faces. You can't restore the original clusters once they've been merged, but you can unassign the (now) wrong faces for some matches, and then manually label a few samples of the other person so that other matching faces are automatically recognized.
